tests/testsuite/generate_lockfile.rs

```diff
@@ -61,15 +60,35 @@ fn adding_and_removing_packages() {
 #[cargo_test]
 fn no_index_update_sparse() {
     let _registry = RegistryBuilder::new().http_index().build();
-    no_index_update();
+    no_index_update(
+        str![[r#"
+[UPDATING] `dummy-registry` index
+[LOCKING] 2 packages to latest compatible versions
+
+"#]],
+        str![[r#"
+[LOCKING] 2 packages to latest compatible versions
+
+"#]],
+    );
 }
 
 #[cargo_test]
 fn no_index_update_git() {
-    no_index_update();
+    no_index_update(
+        str![[r#"
+[UPDATING] `dummy-registry` index
+[LOCKING] 2 packages to latest compatible versions
+
+"#]],
+        str![[r#"
+[LOCKING] 2 packages to latest compatible versions
+
+"#]],
+    );
 }
 
-fn no_index_update() {
+fn no_index_update(e1: Inline, e2: Inline) {
```

Comment:
- Please give more self-describing names
- Generally, we should use `impl IntoData` to not over-prescribe the caller.

Reply:
Naming things so hard! 😅

Reply:
@eth3lbert Would you like to help update other places using `Inline` we merged recently?

Reply:
No problem!